verdaccio/config.yaml

```diff
@@ -26,5 +26,8 @@ packages:
     publish: $authenticated
     unpublish: $authenticated
     proxy: npmjs
+listen:
+  - localhost:4873
```

Comment:
What's the purpose of this?

Reply:
when I was installing my modules locally I had to add this to get the server to respond